app/views/dashboard/protocols/_table.html.haml

```diff
@@ -40,6 +40,8 @@
         %tr
           %th.w-10{ data: { field: 'id', align: 'center', sortable: 'true' } }
             = Protocol.human_attribute_name(:id)
+          %th.w-10{ data: { field: 'protocol_merges', align: 'center', sortable: 'true', visible: 'false' } }
```

Comment:
Could you put the text in YAML (the attribute names are coming from `en.activerecord.models.attributes.<atttribute_name>` so you could define one called `merged_protocols: "Historical Protocol Merges"` or something along those lines